x-pack/test/plugin_api_integration/test_suites/licensed_feature_usage/feature_usage.ts

```diff
@@ -44,6 +44,16 @@ export default function ({ getService }: FtrProviderContext) {
             license_level: 'platinum',
             name: 'Test feature C',
           },
+          {
+            last_used: null,
+            license_level: 'gold',
+            name: 'Subfeature privileges',
+          },
```

Comment:
Sorry about this one, but to avoid all other futur consumers of the service to have to do the same thing, could you adapt the test to only assert existence / date of the 3 `Test feature X` instead?

Reply:
I absolutely can! I wasn't sure about the intent of this test, but your suggestion makes sense to me based on the test description.